src/llama_stack_provider_lmeval/config.py

```diff
@@ -32,6 +32,9 @@ class LMEvalBenchmarkConfig(BenchmarkConfig):
     metadata: Optional[Dict[str, Any]] = None
     # Optional TLS certificate path (or False, to disable TLS verification)
     tls: Optional[Union[str, bool]] = None
+    # Optionally tokenize requests (or False to not tokenize requests)
+    tokenized_requests: Optional[bool] = None
```

Comment:
Should we set a default of `False` and remove the optional? I'm OK with leaving it, but unlike other parameters, this is really either True or False (even though None is Falsey)